roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -58,6 +58,14 @@
   set_fact:
     named_certificates_ca_file_defined: "{{ named_certificates | lib_utils_oo_collect('cafile')}}"
 
+- name: Show message if the cafile is not configured
+  fail:
+    msg: "The cafile is not configured in the openshift_named_certificates. With cafile the cluster's components will trust the named certificate signer."
```

Comment:
Add to this message to skip this check set openshift_named_certificate_omit_cafile=True in the inventory.